src/clients/authclient.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
```

Comment:
rename of client mean also different CLI as you need to run `yast2 authclient` instead of `yast2 auth-client`. What is reason behind it?